cdap-api/src/main/java/co/cask/cdap/api/data/format/StructuredRecord.java

```diff
@@ -56,6 +56,10 @@ public Schema getSchema() {
     return (T) fields.get(fieldName);
   }
 
+  public Map<String, Object> getFields() {
+    return fields;
```

Comment:
Thanks for catching! I wanted to spit out the entire structured record as a json from a ``Transform`` class. I'll make it immutable.